data/are_we_yet/web.md

```diff
@@ -0,0 +1,253 @@
+    packages:
+    - asn1-combinators
+    - avro
+    - capnp
+    - grpc
+    - protobuf
+    - rpc
+    - jsonrpc
+    - thrift
+    - bson
+    - cbor
+    - mqtt
+  - name: Browser
+    status: 🟢
+    description: |
+      OCaml has excellent support for running in the browser!
+      The [Js_of_ocaml](/p/js_of_ocaml/latest) compiler, which translates OCaml into JavaScript, is a well-established tool dating back to 2010. It integrates seamlessly with the existing OCaml ecosystem and powers numerous production applications.
+
+      In addition, [Melange](/p/melange/latest) is a more recent entry to the OCaml-JavaScript compiler space. Originating as a fork of [ReScript](https://rescript-lang.org/), Melange may not have the extensive track record of Js_of_ocaml, but it builds on a very strong foundation and already integrates well with Dune, the OCaml build system.
+    packages:
+    - js_of_ocaml
+    - melange
+    - brr
+    - gen_js_api
+  - name: Compression
+    status: 🟡
+    description: |
+      n/a
+    packages:
+    - zlib
+    - decompress
+    - camlzip
+    - lz4
+    - ezgzip
+    - hpack
+    - brotli
+    - zstd
+    - snappy
+    - lzo
+    - zstandard
+  - name: Crypto
+    status: 🟡
+    description: |
+      n/a
+    packages:
+    - cryptokit
+    - mirage-crypto
+    - nocrypto
+    - pkcs11
+    - tls
+    - hacl_x25519
+    - x509
+    - fiat-p256
+    - sha
+    - blake2
+    - blake3
+    - bls12-381
+    - pbkdf
+    - dirsp-proscript
+  - name: Database
+    status: 🟠
+    description: |
+      n/a
+    packages:
+    - caqti
+    - pgocaml
+    - sqlite3
+    - lmdb
+    - irmin
+    - mysql
+    - postgresql
+    - aws-rds
+    - sequoia
+    - sqlgg
+    - petrol
+    - mariadb
+  - name: Email
+    status: 🟠
+    description: |
+      n/a
+    packages:
+    - tidy_email
+    - emile
+    - smtp
+    - letters
+    - mrmime
+    - sendmail
+    - facteur
+    - received
+    - email_message
+  - name: En- & Decoding
+    status: 🟠
+    description: |
+      n/a
+    packages:
+    - tar
+    - imagelib
+    - base64
+    - multipart-form-data
+    - multipart_form
+    - biniou
+  - name: Web Frameworks
+    status: 🟠
+    description: |
+      n/a
+    packages:
+    - dream
+    - eliom
+    - opium
+    - sihl
+  - name: HTTP Clients
+    status: 🔴
+    description: |
+      n/a
+    packages:
+    - cohttp
+    - hyper
+    - ocurl
+  - name: Internationalization
+    status: 🆘
+    description: |
+      n/a
+    packages:
+    - gettext
+  - name: Logging
+    status: 🟠
+    description: |
+      n/a
+    packages:
+    - logs
+    - dolog
+    - tracing
+    - catapult
+    - opentelemetry
+    - prometheus
+  - name: Lower Web-Stack
+    status: 🟡
+    description: |
+      n/a
+    packages:
```

Comment:
Please adds `paf` and also `mimic` which is like `conduit` and remove `colombe` (which should be move into email stuffs)